### ISSUE TITLE
KWOC 2023 : Airbus Ship Detection

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#625]
Define You:

- [ ] KWOC 2023 Participant 
- [ ] Contributor

Hey OS participants, good to see you.

This project will help us in detecting the airbus ship using the dataset given.

Dataset : https://www.kaggle.com/c/airbus-ship-detection/data

If you like to work on this issue, follow the given guidelines for code as well as for the README. You can try to compare it using different algorithms for better modelling practices..

**Comment by Pranay38** [comment]
Please assign this to me. I want to work on this issue. We can use the YOLO model for ship detection

**Comment by prathimacode-hub** [comment]
Are you an open source contributor or KWOC? @Prakashd20

**Comment by Prakashd20** [comment]
> Are you an open source contributor or KWOC? @Prakashd20 

for delta winter of code!!!

**Comment by prathimacode-hub** [comment]
Ok, issue assigned?

**Comment by Prakashd20** [comment]
@prathimacode-hub ma'am is this project comes under Dwoc'24??

**Comment by prathimacode-hub** [comment]
Yes, it comes under DWOC. @Prakashd20

**Comment by prathimacode-hub** [comment]
@Prakashd20, Have you started working on this issue? I guess before that you need to write a proposal.

**Comment by Prakashd20** [comment]
> @Prakashd20, Have you started working on this issue? I guess before that you need to write a proposal. 

should I submit the proposal for the same till 20th??

**Comment by prathimacode-hub** [comment]
Yeah, do submit the proposal. I will share the details shortly over Discord. @Prakashd20

**Comment by prathimacode-hub** [comment]
@Prakashd20, Are you interested to work on this issue?

**Comment by rnavaneeth992** [comment]
@prathimacode-hub Can I work on this issue

**Comment by prathimacode-hub** [comment]
Yes, sure. Go ahead. @rnavaneeth992

**Comment by Prestigious526** [comment]
@prathimacode-hub I would like to work on this issue.

**Comment by prathimacode-hub** [comment]
Issue assigned to @Prestigious526

**Comment by prathimacode-hub** [comment]
Any update? @Prestigious526

**Comment by prathimacode-hub** [comment]
Hello.. @Prestigious526

**Comment by prathimacode-hub** [comment]
Since there is no update, I have to unassign you from this issue. Kindly be active next time.. @Prestigious526
